contents/terms.mdx

```diff
@@ -185,7 +185,7 @@ Where we process any customer personal data in connection with this Agreement, w
   <summary> PostHog Professional Services Terms </summary>
```

Comment:
Flagging these changes as they may have legal or regulatory requirements to update.